analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/IOperationExtensionsTest.cs

```diff
@@ -32,18 +32,38 @@ public class IOperationExtensionsTest
         [TestMethod]
         public void TrackedSymbol_LocalReference_IsVariableSymbol()
         {
-            var localreference = ((ISimpleAssignmentOperation)TestHelper.CompileCfgBodyCS("var a = true;").Blocks[1].Operations[0]).Target;
-            var symbol = ILocalReferenceOperationWrapper.FromOperation(localreference).Local;
-            localreference.TrackedSymbol().Should().Be(symbol);
+            var localReference = ((ISimpleAssignmentOperation)TestHelper.CompileCfgBodyCS("var a = true;").Blocks[1].Operations[0]).Target;
+            var symbol = ILocalReferenceOperationWrapper.FromOperation(localReference).Local;
+            localReference.TrackedSymbol().Should().Be(symbol);
         }
 
         [TestMethod]
         public void TrackedSymbol_ParameterReference_IsParameterSymbol()
         {
             var expressionStatement = (IExpressionStatementOperation)TestHelper.CompileCfgBodyCS("parameter = true;", "bool parameter").Blocks[1].Operations[0];
-            var paramaterReference = ((ISimpleAssignmentOperation)expressionStatement.Operation).Target;
-            var symbol = IParameterReferenceOperationWrapper.FromOperation(paramaterReference).Parameter;
-            paramaterReference.TrackedSymbol().Should().Be(symbol);
+            var parameterReference = ((ISimpleAssignmentOperation)expressionStatement.Operation).Target;
+            var symbol = IParameterReferenceOperationWrapper.FromOperation(parameterReference).Parameter;
+            parameterReference.TrackedSymbol().Should().Be(symbol);
+        }
+
+        [TestMethod]
+        public void TrackedSymbol_FieldReference_IsFieldSymbol()
```

Comment:
If we merge these two to `DataTestMethod`, it should be cheap to add the static variant as well (naïve assumption that the indexes will match). I'd prefer the static version to be primary here over the `RoslynSymbolicExecutionTest`. Here it's more a UT, there, it's more IT. 
As it should be cheap on both sides, I'd keep it on both places.